modules/apps/portal-background-task/portal-background-task-service/src/main/java/com/liferay/portal/background/task/internal/BackgroundTaskMessagingConfigurator.java

```diff
@@ -42,15 +46,25 @@
  * @author Vendel Toreki
  */
 @Component(
+	configurationPid = "com.liferay.portal.background.task.configuration.BackgroundTaskManagerConfiguration",
 	immediate = true, service = BackgroundTaskMessagingConfigurator.class
 )
 public class BackgroundTaskMessagingConfigurator {
 
 	@Activate
-	protected void activate(BundleContext bundleContext) {
+	protected void activate(
+			BundleContext bundleContext, Map<String, Object> properties)
+		throws PortalException {
+
+		BackgroundTaskManagerConfiguration backgroundTaskManagerConfiguration =
+			ConfigurableUtil.createConfigurable(
```

Comment:
@yuhai This configuration also suffers from not being able to update when configuration is modified.